### PR TITLE
Fix bug for inquiry payment for normal capture

### PR DESCRIPTION
**Pull request by daliwei**
cc @pierre for review. 

Use the actual retry for janitor flow on MarkForCapture because inquiry is not supported on MarkForCapture.

### CODE REVIEW

**Comment by daliwei** [comment]
cc @pierre for review.

File: spec/orbital/remote/shared_examples_for_payment_flow.rb [hunk 1]
```diff
@@ -188,6 +189,59 @@
...
+    @kb_payment.transactions[1].transaction_type = :CAPTURE
+    @kb_payment.transactions[1].amount = @amount
+    @kb_payment.transactions[1].currency = @currency
+
+    capture_properties = merge_properties(@properties, {:trace_number =>  build_random_trace_num, :skip_gw => true})
+    @plugin.capture_payment(@pm.kb_account_id, @kb_payment.id, @kb_payment.transactions[1].id, @pm.kb_payment_method_id, @amount, @currency, capture_properties, @call_context)
+
+    # Force a transition to :UNDEFINED
+    transition_last_response_to_UNDEFINED(2)
+    fix_transaction(1)
```
Comment (pierre) [inline spec/orbital/remote/shared_examples_for_payment_flow.rb:242]: Isn't there a `check_` missing here?

Comment (daliwei) [reply spec/orbital/remote/shared_examples_for_payment_flow.rb:242]: Fixed in d20dc72

**Review by pierre — APPROVED** [review]

**Comment by pierre** [comment]
v0.1.13 released.

**Comment by daliwei** [comment]
Thanks!

**Pull request merged**